pipelines/cpLinearitySolve.yaml

```diff
@@ -0,0 +1,7 @@
+description: cp_pipe linearity calibration construction.
+tasks:
+  linearitySolve:
+    class: lsst.cp.pipe.LinearitySolveTask
+    config:
+      connections.inputPtc: ptc
```

Comment:
This has to match the `dataset_type_name` that is stored in the Butler.  It can technically be anything, as long as it's consistent between the PTC solve code and the downstream linearity/bfk code.  This value matches what we have in https://github.com/lsst/cp_pipe/blob/master/pipelines/measurePhotonTransferCurve.yaml#L39 so I'm hesitant to change it.